src/Backend/Core/Engine/Meta.php

```diff
@@ -625,4 +570,14 @@ public function getData()
     {
         return $this->data;
     }
+
+    /**
+     * @return MetaEntity
+     */
+    public function getMetaEntity()
+    {
+        $this->validate();
+
+        return MetaEntity::fromBackendMeta($this);
```

Comment:
For this time I disagree since this class is now deprecated, so it is just a temporary thing to make your life more easy if you still use SpoonForm but want to use the entity